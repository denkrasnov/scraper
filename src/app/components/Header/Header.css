.header {
  padding-top: 10px;
  width: 100%;
  text-align: center;
  background-color: #fff;
}
