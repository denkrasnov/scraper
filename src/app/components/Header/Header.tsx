import React, { FC } from "react";

import Box from "../../../components/Box";
import Text from "../../../components/Text";

const Header: FC = () => {
  return (
    <Box alignItems="center" height="60px">
      <header>
        <Text color="MAIN_RED" fontSize="fs400" fontWeight="fw700">
          compare
        </Text>
      </header>
    </Box>
  );
};

export default Header;
