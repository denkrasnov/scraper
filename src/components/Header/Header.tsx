import React, { FC } from "react";

import { HeaderProps } from "./types";
import styles from "./Header.css";

const Header: FC<HeaderProps> = props => {
  const { text } = props;
  return (
    <header className={styles.header}>
      <h1>{text}</h1>
    </header>
  );
};

export default Header;
