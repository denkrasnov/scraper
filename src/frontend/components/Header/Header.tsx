import React, { FC } from "react";

import Text from "~app/atoms/Text";
import Box from "~app/atoms/Box";

const Header: FC = () => {
  return (
    <header>
      <Box alignItems="center" height="60px" justifyContent="center">
        <Text color="MAIN_RED" fontSize="fs16" fontWeight="fw700">
          compare
        </Text>
      </Box>
    </header>
  );
};

export default Header;
