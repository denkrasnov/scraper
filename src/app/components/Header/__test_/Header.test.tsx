import React from "react";
import { shallow } from "enzyme";

import Header from "..";

describe("Header", () => {
  it("renders", () => {
    const component = shallow(<Header />);
    expect(component).toMatchSnapshot();
  });
});
