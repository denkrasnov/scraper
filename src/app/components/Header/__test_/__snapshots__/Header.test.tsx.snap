// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders 1`] = `
<Box
  alignItems="center"
  height="60px"
>
  <header>
    <Text
      color="MAIN_RED"
      fontSize="fs400"
      fontWeight="fw700"
    >
      compare
    </Text>
  </header>
</Box>
`;
